src/backend/query_builder.rs

```diff
@@ -637,6 +637,11 @@ pub trait QueryBuilder: QuotedBuilder {
         self.prepare_simple_expr(&order_expr.expr, sql, collector);
         write!(sql, " ").unwrap();
         self.prepare_order(&order_expr.order, sql, collector);
+        match order_expr.nulls_last {
+            Some(true) => write!(sql, " NULLS LAST").unwrap(),
+            Some(false) => write!(sql, " NULLS FISRT").unwrap(),
+            _ => (),
+        }
```

Comment:
This should be placed inside `src/backend/postgres/query.rs`. So, by default all other backend won't write order by nulls when preparing the select statement.